posthog/permissions.py

```diff
@@ -54,6 +54,17 @@ def has_permission(self, request: Request, view) -> bool:
         return settings.MULTI_TENANCY or not User.objects.exists()
 
 
+class IsMultiTenancyAdmin(BasePermission):
+    """
+    Allows access to only staff users on cloud.
+    """
+
+    message = "You are not an admin."
+
+    def has_permission(self, request, view):
+        return not settings.MULTI_TENANCY or request.user.is_staff
```

Comment:
this would be true if `not settings.MULTI_TENANCY`, but the class name and docstring suggest it the condition is 
```
settings.MULTI_TENANCY and request.user.is_staff
```
Perhaps we can name the class `AdminOrNotMultiTenancy` (naming consistency too - looks like we don't use `Is...` for others)?

Reply:
Renamed ;)